src/test/java/com/snowplowanalytics/snowplow/tracker/TrackerTest.java

```diff
@@ -324,36 +343,36 @@ public void testTrackTwoEvents() throws InterruptedException {
         assertEquals(2, results.size());
 
         Map<String, String> result1 = results.get(0).getMap();
-        assertEquals(ImmutableMap.<String, String>builder()
-                .put("dtm", "123456")
+        Map<String, String> expected1 = ImmutableMap.<String, String>builder()
                 .put("ttm", "456789")
                 .put("tz", "Etc/UTC")
                 .put("e", "pv")
                 .put("page", "title")
                 .put("tv", Version.TRACKER)
                 .put("p", "srv")
-                .put("eid", "9783090a-dace-4c85-a75c-933b4596a6c5")
                 .put("tna", "AF003")
                 .put("aid", "cloudfront")
                 .put("refr", "referer")
                 .put("url", "url")
-                .build(), result1);
+                .build();
+
+        assertTrue(result1.entrySet().containsAll(expected1.entrySet()));
 
         Map<String, String> result2 = results.get(1).getMap();
-        assertEquals(ImmutableMap.<String, String>builder()
-                .put("dtm", "123456")
+        Map<String, String> expected2 = ImmutableMap.<String, String>builder()
```

Comment:
It seems the `expected2` is the same as `expected1`. Removing `dtm` and `eid` there aren't differences.
Probably it's the same case for other tests.